ContosoCrafts.WebSite/Models/Category.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+namespace ContosoCrafts.WebSite.Models
+{
+	public class Category
```

Comment:
There is a ProductCategory class inside of the Menu models created that represents the categories json file data. Let's decide on whether we want to refactor it into its own model or if it belongs within menu items model.